packages/ttf_hanazono.rb

```diff
@@ -0,0 +1,29 @@
+require 'package'
+
+class Ttf_hanazono < Package
+  description 'A free Japanese kanji font, which contains about 78,685 characters (and 2 SPACEs) defined in ISO/IEC 10646 standard / the Unicode standard.'
+  homepage 'http://fonts.jp/hanazono' # From ArchLinux PKGBUILD, could not actually resolve site
+  @_ver = 20170904
+  @_revision = 68253
+  version "#{@_ver}-r#{@_revision}"
+  license 'OFL and custom'
+  compatibility 'all'
+  source_url "http://sourceforge.jp/frs/redir.php?m=jaist&f=%2Fhanazono-font%2F#{@_revision}%2Fhanazono-#{@_ver}.zip"
+  source_sha256 '571cd4a09ae7da0c642d640fc2442c050aa450ebb0587a95cdd097d41a9c9572'
+
+  no_compile_needed
+
+  binary_url({
+  })
+
+  binary_sha256({
+  })
```

Comment:
No need to include binary section if `no_compile_needed`.

Reply:
what do you mean?

Reply:
Add the `no_compile_needed` file to the package and the entire `binary_*` section isn't needed.